README.md

```diff
@@ -1,44 +1,29 @@
-## CARO
 
-Common Anatomy Reference Ontology
+![Build Status](https://github.com/obophenotype/caro/workflows/CI/badge.svg)
+# Common Anatomy Reference Ontology
 
-For full details see: [http://obofoundry.org/ontology/caro](http://obofoundry.org/ontology/caro)
+Description: None
 
-This GitHub repo is to manage the source files and release for CARO
+More information can be found at http://obofoundry.org/ontology/caro
 
-## Ontology 
+## Versions
 
-The ontology is stored in [src/ontology/](src/ontology/)
+### Stable release versions
 
-## Bridge Files
+The latest version of the ontology can always be found at:
 
-Bridge files to various AOs are here:
+http://purl.obolibrary.org/obo/caro.owl
 
- * [bridges/](https://github.com/obophenotype/caro/tree/master/src/ontology/bridges)
+(note this will not show up until the request has been approved by obofoundry.org)
 
-## VALIDATION
+### Editors' version
 
-We have a Jenkins job for each anatomy ontology, see the list below:
+Editors of this ontology should use the edit version, [src/ontology/caro-edit.owl](src/ontology/caro-edit.owl)
 
- * [check-caro-fbbt](http://build.berkeleybop.org/job/check-caro-fbbt)
- * [check-caro-po](http://build.berkeleybop.org/job/check-caro-po)
- * [check-caro-xao](http://build.berkeleybop.org/job/check-caro-xao)
- * [check-caro-zfa](http://build.berkeleybop.org/job/check-caro-zfa)
+## Contact
 
-These are all grouped here: http://build.berkeleybop.org/view/Anatomy/
+Please use this GitHub repository's [Issue tracker](https://github.com/obophenotype/caro/issues) to request new terms/classes or report errors or specific concerns related to the ontology.
 
-If an AO has a red ball it means the combination of CARO, the AO and
```

Comment:
Are we sure we want to ditch all of this information in the README?

Reply:
Last we talked, bridges were suppose to stay there but not be used. Don't fully understand any that stuff, but happy to add what is needed in the readme file :)

Reply:
I think this refers to CI that used to run on a Jenkins server at Berkley that tested CARO's compatibility with various bridged ontologies.  We will now do this downstream, but the bridge files may be useful reference in future for activating CARO imports.